src/bflib_render_gpoly.c

```diff
@@ -17,339 +17,360 @@
  *     (at your option) any later version.
  */
 /******************************************************************************/
-#include "pre_inc.h"
-#include "bflib_render.h"
-
-#include "globals.h"
 #include "bflib_basics.h"
 #include "bflib_memory.h"
-#include "bflib_video.h"
+#include "bflib_render.h"
 #include "bflib_sprite.h"
+#include "bflib_video.h"
 #include "bflib_vidraw.h"
+#include "globals.h"
 #include "post_inc.h"
+#include "pre_inc.h"
```

Comment:
I believe this one needs to be the first one, alphabet be damned.

Reply:
I did a `clang-format` on the file so that must be how it got re-ordered. Maybe this is why it is broken currently :)

Anyway, will try to keep it in that order